ext/civicrm_admin_ui/CRM/CivicrmAdminUi/Upgrader.php

```diff
@@ -0,0 +1,41 @@
+<?php
+// phpcs:disable
+use CRM_CivicrmAdminUi_ExtensionUtil as E;
+// phpcs:enable
+
+/**
+ * Collection of upgrade steps.
+ */
+class CRM_CivicrmAdminUi_Upgrader extends CRM_Extension_Upgrader_Base {
+
+  protected function replaceFindContactMenuPath($path) {
+    // point Find Contacts menu to the FB/SK version or back to the original path
+    // this is temporary until everything is in FB/SK and we can use the original path
+    $results = \Civi\Api4\Navigation::update(FALSE)
+      ->addValue('url', $path)
+      ->addWhere('name', '=', 'Find Contacts')
+      ->execute();
+  }
+
+  public function install(): void {
```

Comment:
Do we need install and uninstall? Doesn't enable and disable happen before these?

Reply:
In the [hook documentation](https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_enable/), it seems to suggest that installing doesn't go through the enable:

> It is NOT called when the status changes from uninstalled to enabled

 I assumed it was the same if you uninstall without disabling (although it's not possible from the UI).

Reply:
I think that's different for the hook, this is not the hook. I tested it out via UI and cv and always get both if doing install or uninstall from enabled state.

Reply:
@larssandergreen How about for now we just add a comment and then I can merge it in using this suggest feature?

```suggestion
  /**
   * @todo "install" and "uninstall" may not be needed if enable and disable are present. See https://github.com/civicrm/civicrm-core/pull/26669
   */
  public function install(): void {
```